virttest/utils_version.py

```diff
@@ -28,8 +28,10 @@ def __init__(self, interval):
             raise ValueError("Invaild string representation of an interval")
         self.opening, lower, upper, self.closing = match.groups()
 
-        self.lower_bound = LooseVersion(lower) if lower else LooseVersion("0")
-        self.upper_bound = LooseVersion(upper) if upper else LooseVersion("z")
+        self.lower_bound = LooseVersion(lower) if lower else None
+        self.upper_bound = LooseVersion(upper) if upper else None
+        if self.upper_bound or self.lower_bound:
+            return
```

Comment:
It does need a `ValueError` check when both have `self.upper_bound` and `self.lower_bound`.

Reply:
Do you refer to both lower boundary and upper boundary are None?